components/table/TableData.jsx

```diff
@@ -69,7 +69,9 @@ const TableData = ({
             return (
                 <tr
                     key={uniqueId()}
+                    tabIndex={0}
                     onClick={() => localClickHandler(row, i)}
+                    onKeyPress={() => localClickHandler(row, i)}
```

Comment:
the drilldown should only work on tab + enter not just any keypress so there needs to be a check if it's the enter key.

Reply:
Fixed.